pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -319,7 +319,7 @@ func outputKubeadmInitSteps(logs io.Reader) {
 }
 
 // applyCNI applies CNI to a cluster. Needs to be done every time a VM is powered up.
-func (k *Bootstrapper) applyCNI(cfg config.ClusterConfig) error {
+func (k *Bootstrapper) applyCNI(cfg config.ClusterConfig, isInit bool) error {
```

Comment:
how about rename it to `registerStep bool`

Reply:
make this argument optional
so other callers can just called it

applyCNI
and if not True then it wont register step